src/components/side/Side.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import Stegindikator from 'nav-frontend-stegindikator/lib/stegindikator';
+import TilbakeKnapp from '../TilbakeKnapp';
+import Banner from '../Banner';
+import { Panel } from 'nav-frontend-paneler';
+import NavKnapp, { knapptype } from '../NavKnapp';
+import { useLocation } from 'react-router-dom';
+import { Systemtittel } from 'nav-frontend-typografi';
+import { Routes } from '../../config/Routes';
+import { hentForrigeRoute } from '../../utils/routing';
+
+interface ISide {
+  tittel: string;
+  tilbakePath?: string;
+  nestePath?: string;
+}
+
+const Side: React.FC<ISide> = ({ tittel, nestePath, children }) => {
+  const location = useLocation();
+
+  let stegobjekter = [];
+  const routes = Routes;
+  const aktivtSteg = routes.findIndex(
+    (steg) => steg.path === location.pathname
+  );
+
+  for (let i = 0; i < routes.length; i++) {
```

Comment:
Her kan du kanskje bruke Object.values på routes og mappe til steg som stegindikatoren forventer?

Reply:
Går kanskje an å se på kontantstøtte sin stegindikator også, samme type problemer er løst der.

Reply:
Det tenkte jeg ikke på! Har oppdatert koden! :) Takk for feedback ✨